src/app/components/event-item/controls/fireside-post/extra/extra.ts

```diff
@@ -138,4 +183,24 @@ export default class AppEventItemControlsFiresidePostExtra extends Vue {
 			this.emitRemove();
 		}
 	}
+
+	getPinTargetModel() {
+		let targetModel = 'User';
+		if (this.post.game instanceof Game) {
+			targetModel = 'Game';
```

Comment:
depends on the route, can't assume 'Game'.

Reply:
Fixed.